docs/source/advanced_features/configuration.rst

```diff
@@ -0,0 +1,46 @@
+#######################
+ Request Configuration
+#######################
+
+TensorBay provides lots of convenient services. The default setting of TensorBay service should be
```

Comment:
This topic introduces the currently supported :class:`~tensorbay.client.requests.Config` options(a table link) for customizing request. Note that the default settings can satisfy most use cases.

Delete the first sentence.

![image](https://user-images.githubusercontent.com/32992355/116365833-3c67c280-a838-11eb-9476-a3fc2cd065cc.png)